Framework/Indexing/src/IndexInfo.cpp

```diff
@@ -244,86 +223,58 @@ IndexInfo::globalSpectrumIndicesFromDetectorIndices(const std::vector<size_t> &d
   }
```

Comment:
It has been a long time since I wrote this, but I have a hunch that this entire method (`globalSpectrumIndicesFromDetectorIndices`) should not be necessary any more without MPI. Can you check if it can be removed? Maybe `MatrixWorkspace` has an equivalent (in the non-MPI case) method?

But I may be misremembering, and this may be a faster variant, or required for time-dependent detectors?

Reply:
It is only used in `CropToComponent`. It was added [here](https://github.com/mantidproject/mantid/pull/21170) before adding the MPI code. What do you think?

Reply:
No, this is fine. 👍